lib/methods/trustedDomains.js

```diff
@@ -3,10 +3,11 @@ var BaseMethod = require('./basemethod');
 var ApiKeysMethods = {
     get: BaseMethod.extend({
         requiredParams: [
-            'application_id'
+            'application_id',
+            'page'
         ],
         dataFunction: function(data, cb) {
-            this.path = data['application_id'] + '/trusted_domains';
+            this.path = data['application_id'] + '/trusted_domains?page=' + data['page'];
```

Comment:
Is this going to work with the conversion to use offset and limit? I suspect not.